### DISCUSSION TITLE
Different skins on different pages and posts

### DISCUSSION THREAD

**Discussion by kalaniwilliams** [mmistakes/minimal-mistakes#2695] [Q&A] [answered]
Is it possible currently to use different skins on different pages and posts? I'd like to be able to specify the skin in the front matter on individual posts, but I don't think that's currently possible. Thanks for considering!

**Comment by mmistakes** [comment, marked as answer]
This is not something I'd consider supporting in the theme, but if you're adventurous it is something that *should* be possible. Off the top of my head you'd need to:

1. Create stylesheets for each of the skins. These are currently[ Sass partials](https://github.com/mmistakes/minimal-mistakes/tree/master/_sass/minimal-mistakes/skins) now so you'd need to modify things slightly to get them to build as `.css` files you can reference in a page. 
2. Create new skin stylesheets in `/assets/css/` similar to [`main.scss`](https://github.com/mmistakes/minimal-mistakes/blob/master/assets/css/main.scss) and change the `@import` line that brings in a skin partial to a specific file. For example if you wanted to create a dark stylesheet you'd create `/assets/css/dark.scss` and change line 7 to `@import "minimal-mistakes/skins/dark";`
3. Modify [`_includes/head.html`](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/head.html#L15) to have some logic that if you assign a skin in a page's front matter it uses that stylesheet instead of `main.css`. Something like this perhaps:
     ```html
     {% assign stylesheet = page.skin | default: 'main' | prepend: '/assets/css/' | append: '.css' %}
     <link rel="stylesheet" href="{{ stylesheet | relative_url }}">
     ```
